tests/ga/test_extension.py

```diff
@@ -666,6 +700,126 @@ def test_ext_handler_no_reporting_status(self, *args):
         self._assert_handler_status(protocol.report_vm_status, "Ready", 1, "1.0.0")
         self._assert_ext_status(protocol.report_ext_status, "error", 0)
 
+    def test_wait_for_handler_successful_completion_empty_exts(self, *args):
+        test_data = WireProtocolData(DATA_FILE)
+        exthandlers_handler, protocol = self._create_mock(test_data, *args)
+
+        handler = ExtHandler(name="handler")
+
+        ExtHandlerInstance.get_ext_handling_status = MagicMock(return_value=None)
+        self.assertTrue(exthandlers_handler.wait_for_handler_successful_completion(handler, datetime.datetime.utcnow()))
+
+    def _test_wait_for_handler_successful_completion(self, exthandlers_handler):
```

Comment:
let's remove "test" from the name of this helper function. this threw me off... gave me the impression that it was a test